pkg/devicemapper/devmapper.go

```diff
@@ -264,11 +264,6 @@ func UdevWait(cookie *uint) error {
 	return nil
 }
 
-// LogInitVerbose is an interface to initialize the verbose logger for the device mapper library.
-func LogInitVerbose(level int) {
```

Comment:
> When you say we, you mean docker.

No, I'm not talking about Docker, I'm talking about any potential user of this `pkg`.

`LogInitVerbose` *literally does not mean anything outside of the default `libdm` logger*, and because we (or any user of this package) **need** to set our own callbacks in order to get `errBusy` (which we use internally for a bunch of things) and so on, exposing this API is confusing and pointless. If you register your own logging callback, you get all messages without any filter.

If believe I'm mistaken, you can look at the source of ``:

```c
void dm_log_init_verbose(int level)
{
	_verbose = level;
}
```

And `_verbose` is only used in the default logger:

```
% git grep '\b_verbose\b' libdm/
libdm/libdm-common.c:static int _verbose = 0;
libdm/libdm-common.c:   if (level <= _LOG_WARN || _verbose) {
libdm/libdm-common.c:   _verbose = level;
```

```c
/*
 * Library users can provide their own logging
 * function.
 */

__attribute__((format(printf, 5, 0)))
static void _default_log_line(int level,
	    const char *file __attribute__((unused)),
	    int line __attribute__((unused)), int dm_errno_or_class,
	    const char *f, va_list ap)
{
	static int _abort_on_internal_errors = -1;
	FILE *out = log_stderr(level) ? stderr : stdout;

	level = log_level(level);

	if (level <= _LOG_WARN || _verbose) {
		if (level < _LOG_WARN)
			out = stderr;
		vfprintf(out, f, ap);
		fputc('\n', out);
	}

	if (_abort_on_internal_errors < 0)
		/* Set when env DM_ABORT_ON_INTERNAL_ERRORS is not "0" */
		_abort_on_internal_errors =
			strcmp(getenv("DM_ABORT_ON_INTERNAL_ERRORS") ? : "0", "0");

	if (_abort_on_internal_errors &&
	    !strncmp(f, INTERNAL_ERROR, sizeof(INTERNAL_ERROR) - 1))
		abort();
}
```

And when you register your own logger, it replaces the default one. I checked all of this before removing this API, because I was also confused why it was present in this library. If anyone is using it, it's acting as a noop.